apps/Gruntfile.js

```diff
@@ -2,9 +2,31 @@ var path = require('path');
 var fs = require('fs');
 var mkdirp = require('mkdirp');
 var glob = require('glob');
+var execSync = require('child_process').execSync;
+var write = process.stdout.write.bind(process.stdout);
+
+function log(str) {
+	write(str + '\n', 'utf8');
+}
 
 module.exports = function (grunt) {
-  require('time-grunt')(grunt);
+  var email = 'unknown';
+  try {
+    email = execSync('git config --get user.email').toString().trim();
+  } catch (e) {
+    // I guess we are not in a git checkout, or don't have git installed.
+  }
+  require('time-grunt-nowatch')(grunt, false, function (stats, done) {
```

Comment:
It was previously required here, so I thought I would just leave it as is. But it can be changed!